SerialPrograms/Source/PokemonSV/Programs/PokemonSV_Navigation.cpp

```diff
@@ -398,6 +399,120 @@ void leave_box_system_to_overworld(const ProgramInfo& info, ConsoleHandle& conso
 }
 
 
+void open_pokedex_from_overworld(const ProgramInfo& info, ConsoleHandle& console, BotBaseContext& context){
+    {
+        OverworldWatcher overworld(COLOR_CYAN);
+        context.wait_for_all_requests();
+        int ret = wait_until(
+            console, context,
+            std::chrono::seconds(10),
+            {overworld}
+        );
+        context.wait_for(std::chrono::milliseconds(100));
+        if (ret == 0){
+            console.log("Detected overworld.");
+            pbf_press_button(context, BUTTON_MINUS, 20, 105); // open pokedex

```

Comment:
you are already doing this if you detect overworldin the next detector. Is there any reason not to just remove those 2 lines ?